app/main.py

```diff
@@ -14,12 +14,38 @@
 from sdx.common.logger_config import logger_initial_config
 import tornado.ioloop
 import tornado.web
+from tornado.httpclient import HTTPClient, HTTPError
 
 from encrypter import Encrypter
 from ftpclient import FTPWorker
 from publisher import DurableTopicPublisher
 
 
+class HealthCheckService(tornado.web.RequestHandler):
+
+    def initialize(self, services):
+        self.ftp = FTPWorker(**Task.ftp_params(services))
+        self.amqp_params = Task.amqp_params(services)
+
+    def get(self):
+        http_client = HTTPClient()
+        try:
+            http_client.fetch(self.ampq_params["check"])
+        except HTTPError:
+            # HTTPError is raised for non-200 responses
+            self.send_error(410)
+        except Exception:
+            # Possible IOError, etc
+            self.send_error(410)
+        finally:
+            http_client.close()
+
+        if self.ftp.check():
+            self.set_status(200)
+        else:
+            self.send_error(410)
+
```

Comment:
This health check should be returning rabbitmq and ftp status. For rabbitmq status it will always return a 200 as long as it can be reached in order to check it is ok the response body has to be queried for an ok response. John put an example of how he would like the response from the endpoint on the consumer service enhancements card.